analyzer/testdata/src/checkers-default/empty/empty_test.go

```diff
@@ -9,109 +9,499 @@ import (
+		assert.Lessf(t, len([]string{"e"}), 1, "msg with args %d %s", 42, "42")           // want "empty: use assert\\.Emptyf"
+		assert.Greater(t, 1, len(elems))                                                  // want "empty: use assert\\.Empty"
+		assert.Greaterf(t, 1, len(elems), "msg with args %d %s", 42, "42")                // want "empty: use assert\\.Emptyf"
+		assert.Greater(t, 1, len(str))                                                    // want "empty: use assert\\.Empty"
+		assert.Greaterf(t, 1, len(str), "msg with args %d %s", 42, "42")                  // want "empty: use assert\\.Emptyf"
+		assert.Greater(t, 1, len(string(str)))                                            // want "empty: use assert\\.Empty"
+		assert.Greaterf(t, 1, len(string(str)), "msg with args %d %s", 42, "42")          // want "empty: use assert\\.Emptyf"
+		assert.Greater(t, 1, len(b))                                                      // want "empty: use assert\\.Empty"
+		assert.Greaterf(t, 1, len(b), "msg with args %d %s", 42, "42")                    // want "empty: use assert\\.Emptyf"
+		assert.Greater(t, 1, len(string(b)))                                              // want "empty: use assert\\.Empty"
+		assert.Greaterf(t, 1, len(string(b)), "msg with args %d %s", 42, "42")            // want "empty: use assert\\.Emptyf"
+		assert.Greater(t, 1, len([]string{"e"}))                                          // want "empty: use assert\\.Empty"
+		assert.Greaterf(t, 1, len([]string{"e"}), "msg with args %d %s", 42, "42")        // want "empty: use assert\\.Emptyf"
+		assert.Equal(t, "", elems)                                                        // want "empty: use assert\\.Empty"
+		assert.Equalf(t, "", elems, "msg with args %d %s", 42, "42")                      // want "empty: use assert\\.Emptyf"
+		assert.Equal(t, "", str)                                                          // want "empty: use assert\\.Empty"
+		assert.Equalf(t, "", str, "msg with args %d %s", 42, "42")                        // want "empty: use assert\\.Emptyf"
+		assert.Equal(t, "", string(str))                                                  // want "empty: use assert\\.Empty"
+		assert.Equalf(t, "", string(str), "msg with args %d %s", 42, "42")                // want "empty: use assert\\.Emptyf"
+		assert.Equal(t, "", b)                                                            // want "empty: use assert\\.Empty"
+		assert.Equalf(t, "", b, "msg with args %d %s", 42, "42")                          // want "empty: use assert\\.Emptyf"
+		assert.Equal(t, "", string(b))                                                    // want "empty: use assert\\.Empty"
+		assert.Equalf(t, "", string(b), "msg with args %d %s", 42, "42")                  // want "empty: use assert\\.Emptyf"
+		assert.Equal(t, "", []string{"e"})                                                // want "empty: use assert\\.Empty"
```

Comment:
I would have understood

```suggestion
		assert.Equal(t, "", []byte{"e"})                                                // want "empty: use assert\\.Empty"
```
Here I don't get the comparison between a []string and ""

Reply:
`[]string{"e"}` is a just example of complex structure (brackets, quotes, etc.)
and want to be sure that this is rendered right after fix